planner/core/explain.go

```diff
@@ -172,6 +172,10 @@ func (p *PhysicalSelection) ExplainInfo() string {
 
 // ExplainInfo implements Plan interface.
 func (p *PhysicalProjection) ExplainInfo() string {
+	if p.Concurrency > 0 {
```

Comment:
I remember that the concurrency of projection worker is also controlled by row count, see https://github.com/pingcap/tidb/blob/master/executor/builder.go#L1194. Do we need to consider this?

Reply:
Yes, we need to record the concurrency information of the operator during the execution phase. Update. PTAL. @lamxTyler